DIRECTION_v5.md

```diff
@@ -0,0 +1,134 @@
+4. The `irsa` and `helm-addon` modules will be removed entirely; we have released a new external module [`terraform-aws-eks-addon`](https://github.com/aws-ia/terraform-aws-eks-addon) that is available on the Terraform registry that replicates/replaces the functionality of these two modules. This will now allow users, as well as partners, to create their own addons that are not natively supported by EKS Blueprints more easily and following the same process as EKS Blueprints.
+5. The `aws-eks-teams` module will be removed entirely; its replacement will be the new external module [`terraform-aws-eks-teams`](#TODO) that incorporates the changes customers have been asking for in https://github.com/aws-ia/terraform-aws-eks-blueprints/issues/842
+
+### Resulting Project Structure
+
+Previously under the v4.x structure, the EKS Blueprint project was comprised of various repositories across multiple AWS organizations that looked roughly like the following:
+
+#### v4.x Structure
+
+```
+├── aws-quickstart/
+|   └── cdk-eks-blueprints/
+|       └── examples/
+├── aws-ia/
+|   ├── ecs-blueprints/
+|   |   └── examples/
+|   ├── terraform-aws-eks-blueprints/
+|   |   ├── aws-auth-configmap.tf
+|   |   ├── data.tf
+|   |   ├── eks-worker.tf
+|   |   ├── locals.tf
+|   |   ├── main.tf
+|   |   ├── outputs.tf
+|   |   ├── variables.tf
+|   |   ├── versions.tf
+|   |   ├── examples/
+|   |   └── modules
+|   |       ├── aws-eks-fargate-profiles/
+|   |       ├── aws-eks-managed-node-groups/
+|   |       ├── aws-eks-self-managed-node-groups/
+|   |       ├── aws-eks-teams/
+|   |       ├── aws-kms/
+|   |       ├── emr-on-eks/
+|   |       ├── irsa/
+|   |       ├── kubernetes-addons/
+|   |       └── launch-templates/
+|   └── terraform-aws-eks-ack-addons/
+├── awslabs/
+|   ├── crossplane-on-eks/
+|   └── data-on-eks/
+├── aws-samples/
+|   ├── eks-blueprints-add-ons/
+|   └── eks-blueprints-workloads/
+└── aws-observability/
+    └── terraform-aws-observability-accelerator/
+```
+
+#### v5.x Structure
+
+```
+├── aws-quickstart/
+|   └── cdk-eks-blueprints/
+|       └── examples/
+├── aws-ia/
+|   ├── ecs-blueprints/
+|   |   └── examples/
+|   ├── terraform-aws-eks-blueprints/
+|   |   └── examples/
+|   ├── eks-addons/                      # Contains addons in supported formats (Terraform, ArgoCD, and FluxCD (future))
+|   ├── terraform-aws-eks-addon/         # Module for creating Terraform based addon (IRSA + Helm chart)
+|   ├── terraform-aws-eks-ack-addons/    # Will contain addon support for multiple frameworks in one location
+|   |   └── modules/
+|   |       ├── aws-load-balancer-controller/
+|   |       |   ├── main.tf
+|   |       |   ├── outputs.tf
+|   |       |   ├── variables.tf
+|   |       |   └── versions.tf
+|   |       └── <other-addons>/
+|   ├── terraform-aws-eks-multi-tenancy/ # Module for creating Kubernetes multi-tenancy constructs
```

Comment:
Is this `terraform-aws-eks-multi-tenancy` same as `terraform-aws-eks-teams`?

Reply:
name updated in https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/1319/commits/0a8023866746c8be15c587b283c4090c8254d63d